app/views/vacancies/index.html+phone.haml

```diff
@@ -4,24 +4,22 @@
   = content_for :page_title_prefix, t('jobs.heading')
 = render partial: 'head_links'
 
-%h1.govuk-heading-xl
-  = t('jobs.heading')
+%h1.govuk-heading-xl#job-count
+  = @vacancies_search.location_category_search? ? @vacancies.total_count_message_with_location(format_location_name(@vacancies_search.location_category)) : @vacancies.total_count_message
 .govuk-grid-row
   .govuk-grid-column-one-third
     = render 'filters', collapsible?: true
 
   .govuk-grid-column-two-thirds
     .govuk-grid-column-full.vacancies-count
-      %p.govuk-heading-l.mb0= @vacancies_search.location_category_search? ? @vacancies.total_count_message_with_location(format_location_name(@vacancies_search.location_category)) : @vacancies.total_count_message
       - if @vacancies.user_search?
-        %span.clear-search.nobreak= link_to t('jobs.filters.clear_filters'), root_path, class: 'govuk-link'
         = render(partial: 'subscribe_link', locals: { search_criteria: @vacancies_search.only_active_to_hash }) if EmailAlertsFeature.enabled? && !ReadOnlyFeature.enabled?
     - if @vacancies.any?
       = render partial: 'sorting_options', locals: { search_criteria: @vacancies_search.only_active_to_hash, sort: @vacancies_search.sort_by }
-
-      %ul.vacancies
-        - @vacancies.each do |vacancy|
-          = render partial: 'vacancy', locals: { vacancy: vacancy }
+      #vacancies-hits
```

Comment:
Do we need the extra div or can we just add the ID onto the `%ul` element?

Reply:
i think you should have a container element, not  going to change this right now as it breaks the page anyway